test/test_moves_files_to_local_machine.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+# You can run it from any directory.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Yup, but the problem is that I can't extract this to `common.sh` because to include `common.sh` I need to know directory where the script is in a way that does not depend on from where was the script launched…